src/devices/video/ef9345.cpp

```diff
@@ -239,7 +257,17 @@ void ef9345_device::draw_char_80(uint8_t *c, uint16_t x, uint16_t y)
 // set then ef9345 mode
 void ef9345_device::set_video_mode(void)
 {
-	m_char_mode = ((m_pat & 0x80) >> 5) | ((m_tgs & 0xc0) >> 6);
+	if(m_variant == TYPE_TS9347)
```

Comment:
This doesn't follow the spacing of the rest of the function - the rest of the file has space between the `if` keyword and opening parenthesis of the expression.